fhir-bulkdata-webapp/src/main/resources/META-INF/batch-jobs/FhirBulkExportFastJob.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<job xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://xmlns.jcp.org/xml/ns/javaee" xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/jobXML_1_0.xsd" id="bulkexportfastjob" restartable="true" version="1.0">
+    <listeners>
+        <listener ref="com.ibm.fhir.bulkdata.jbatch.export.fast.ExportJobListener">
+            <properties>
+                <property name="fhir.tenant" value="#{jobParameters['fhir.tenant']}"/>
+                <property name="fhir.datastoreid" value="#{jobParameters['fhir.datastoreid']}"/>
+                <property name="fhir.incomingUrl" value="#{jobParameters['fhir.incomingUrl']}"/>
+                <property name="fhir.bulkdata.source" value="#{jobParameters['fhir.bulkdata.source']}"/>
+                <property name="fhir.bulkdata.outcome" value="#{jobParameters['fhir.bulkdata.outcome']}"/>
+            </properties>
+        </listener>
+    </listeners>
+    <step id="step1">
+        <listeners>
+            <listener ref="com.ibm.fhir.bulkdata.jbatch.listener.StepChunkListener"></listener>
+        </listeners>
+        <chunk checkpoint-policy="custom">
+            <reader ref="com.ibm.fhir.bulkdata.jbatch.export.fast.ResourcePayloadReader">
+                <properties >
+                    <property name="fhir.tenant" value="#{jobParameters['fhir.tenant']}"/>
+                    <property name="fhir.datastoreid" value="#{jobParameters['fhir.datastoreid']}"/>
+                    <property name="fhir.incomingUrl" value="#{jobParameters['fhir.incomingUrl']}"/>
+                    <property name="fhir.bulkdata.source" value="#{jobParameters['fhir.bulkdata.source']}"/>
+                    <property name="fhir.bulkdata.outcome" value="#{jobParameters['fhir.bulkdata.outcome']}"/>
+                    
+                    <property name="cos.bucket.pathprefix" value="#{jobParameters['cos.bucket.pathprefix']}"/>
+                    <property name="partition.resourcetype" value="#{partitionPlan['partition.resourcetype']}"/>
+                    <property name="fhir.exportFormat" value="#{jobParameters['fhir.exportFormat']}"/>
+                    <property name="fhir.search.fromdate" value="#{jobParameters['fhir.search.fromdate']}"/>
+                    <property name="fhir.search.todate" value="#{jobParameters['fhir.search.todate']}"/>
+                    <property name="fhir.typeFilters" value="#{jobParameters['fhir.typeFilters']}"/>
+                </properties>
+            </reader>
+            <writer ref="com.ibm.fhir.bulkdata.jbatch.export.fast.NopWriter">
+                <properties>
+                </properties>
+            </writer>
+            <checkpoint-algorithm ref="com.ibm.fhir.bulkdata.jbatch.export.fast.checkpoint.ResourceExportCheckpointAlgorithm">
```

Comment:
The class ref doesn't match its updated package name.
```suggestion
            <checkpoint-algorithm ref="com.ibm.fhir.bulkdata.export.fast.checkpoint.ResourceExportCheckpointAlgorithm">
```

Reply:
merged locally